src/Entity/Game.php

```diff
@@ -37,6 +37,11 @@ class Game
      */
     private $publisher;
 
+    /**
+     * @ORM\Column(type="boolean", options={"default":0}, nullable=true)
```

Comment:
If `` options={"default":0} `` did not work for you, then just delete it, since assigning a default value for the property does the same thing. Maybe `` "default":"false"}  `` would do the trick here, but I am unsure and can't test as it is now, so lets just leave it with a default value.